test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/ValidateDuplicateTransactionAfterReconnect.java

```diff
@@ -62,7 +63,10 @@ private HapiApiSpec validateDuplicateTransactionAfterReconnect() {
 						sleepFor(Duration.ofSeconds(25).toMillis()),
 						getAccountBalance(GENESIS)
 								.setNode("0.0.6")
-								.unavailableNode()
+								.unavailableNode(),
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(ADDRESS_BOOK_CONTROL)
+								.overridingProps(Map.of("ledger.keepRecordsInState", "true"))
```

Comment:
Isn't `ledger.keepRecordsInState` in `bootstrap.properties` instead of `application.properties`? I wonder if this property is dynamically updated via a fileUpdate of `application.properties`. @tinker-michaelj, could you please confirm?

Reply:
@qnswirlds When discussed with @tinker-michaelj, though `ledger.keepRecordsInState` shows its set in `bootstrap.properties` in `hgcaa.log` , the default value is taken from Global/Dynamic properties , so we can override in `application.properties`.